napari/_tests/test_draw.py

```diff
@@ -10,10 +10,12 @@
 )
 def test_canvas_drawing(make_napari_viewer):
     """Test drawing before and after adding and then deleting a layer."""
-    viewer = make_napari_viewer()
+    viewer = make_napari_viewer(show=True)
```

Comment:
This fixes the test locally and on CI 🤷🏼  @sofroniewn

Reply:
this is fine but add the same skip condition that we use in `test_with_screenshot`
```
skip_local_popups = pytest.mark.skipif(
    not os.getenv('CI') and os.getenv('NAPARI_POPUP_TESTS', '0') == '0',
    reason='Tests requiring GUI windows are skipped locally by default.',
)
```